### PR TITLE
filter slates

### PR DESCRIPTION
**Pull request by kangarang**
if on rinkeby network, filter out old slates and substitute organization field of slate 5

### CODE REVIEW

File: api/utils/slates.js [hunk 1]
```diff
@@ -49,8 +49,17 @@ async function getAllSlates() {
   const slateIDs = range(0, slateCount);
   console.log('slateIDs', slateIDs);
 
+  // TEMPORARY HACK
+  let slateIDsToQuery = slateIDs;
+  const network = await provider.getNetwork();
+  if (network.chainId === 4) {
+    const garbage = [0, 1, 2, 3];
```
Comment (akuanti) [inline api/utils/slates.js]: I think we want to filter to keep slateIDs in `[4, 6]` rather than the inverse.

File: api/utils/slates.js [hunk 2]
```diff
@@ -114,6 +137,11 @@ async function getSlateWithMetadata(slateID, slate, metadataHash, incumbent, req
     console.log('proposalMultihashes:', proposalMultihashes);
     console.log('');
 
+    // TEMPORARY HACK
+    if (chainId === 4 && slateID === 5) {
```
Comment (akuanti) [inline api/utils/slates.js]: We don't need to show slate 5, so you don't need this.

File: api/utils/slates.js (hunk 1 above)
Comment (kangarang) [reply api/utils/slates.js]: im gonna do `!Array.includes` in case we add more slates

File: api/utils/slates.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ async function getAllSlates() {
   let slateIDsToQuery = slateIDs;
   const network = await provider.getNetwork();
   if (network.chainId === 4) {
-    const garbage = [0, 1, 2, 3];
+    const garbage = [0, 1, 2, 3, 5];
```
Comment (akuanti) [inline api/utils/slates.js]: I think it's probably better to filter for the ones we want to show rather than the ones we don't, but this will do for now.

**Pull request merged**